contributing/CHANGES.md

```diff
@@ -1,8 +1,8 @@
-# Recording user facing changes in CHANGES.md
```

Comment:
[question] Is this file's name remaining the same?

Reply:
If that seems ok? Was thinking it referred to the 'first' use of changes, rather than the second.